plane/schema/migrations/20231027092522_initial-schema.sql

```diff
@@ -91,17 +91,20 @@ create index idx_backend_action_pending on backend_action(drone_id) where acked_
 create index idx_backend_action_backend on backend_action(backend_id);
 
 create table backend_key (
-    id serial primary key,
+    id varchar(255) references backend(id) not null,
```

Comment:
Do we need to designate this as a primary key? Is there any uniqueness constraint right now that makes sure a backend only has one lock?